openfga_sdk/sync/open_fga_api.py

```diff
@@ -10,7 +10,6 @@
    NOTE: This file was auto generated by OpenAPI Generator (https://openapi-generator.tech). DO NOT EDIT.
 """
 
-
```

Comment:
This is necessary to make the linter happy and pass the tests. I will check why it's catching it here, but the Generator output isn't.